### PR TITLE
Fix NumberBox's APITests project location in the innerloop solution.

### PR DESCRIPTION
**Pull request by Felix-Dev**
I guess third time's the charm...after PRs https://github.com/microsoft/microsoft-ui-xaml/pull/2981 and https://github.com/microsoft/microsoft-ui-xaml/pull/2224 the NumberBox's APITests project was still not correctly placed in the inner loop solution:

image

Expected position:

image

This PR should finally get the job properly done. Here's the position of the APITests project following this PR:

image

### CODE REVIEW

File: MUXControlsInnerLoop.sln [hunk 1]
```diff
@@ -568,7 +568,6 @@ Global
 		dev\CommonStyles\TestUI\CommonStyles_TestUI.projitems*{a7f6d6c4-a5a9-43eb-930c-b766417a5e5c}*SharedItemsImports = 13
 		dev\Materials\Acrylic\TestUI\AcrylicBrush_TestUI.projitems*{a800e818-7212-4fd7-ae3a-1dcab539db87}*SharedItemsImports = 13
 		dev\Collections\Collections.vcxitems*{ad0c90b0-4845-4d4b-88f1-86f653f8171b}*SharedItemsImports = 4
-		dev\ComboBox\ComboBox.vcxitems*{ad0c90b0-4845-4d4b-88f1-86f653f8171b}*SharedItemsImports = 4
```
Comment (Felix-Dev) [inline MUXControlsInnerLoop.sln:571]: I have no real clue how these changes managed to sneak into this file here...

Comment (chingucoding) [reply MUXControlsInnerLoop.sln:571]: Neither does VS I think.

File: MUXControlsInnerLoop.sln [hunk 2]
```diff
@@ -577,9 +576,6 @@ Global
 		dev\Lights\Lights.vcxitems*{ad0c90b0-4845-4d4b-88f1-86f653f8171b}*SharedItemsImports = 4
 		dev\Materials\Acrylic\AcrylicBrush.vcxitems*{ad0c90b0-4845-4d4b-88f1-86f653f8171b}*SharedItemsImports = 4
 		dev\Materials\Reveal\RevealBrush.vcxitems*{ad0c90b0-4845-4d4b-88f1-86f653f8171b}*SharedItemsImports = 4
-		dev\NumberBox\NumberBox.vcxitems*{ad0c90b0-4845-4d4b-88f1-86f653f8171b}*SharedItemsImports = 4
```
Comment (Felix-Dev) [inline MUXControlsInnerLoop.sln:580]: Same as above. I assume they just slipped through with another commit but don't really belong in here.

File: MUXControlsInnerLoop.sln [hunk 3]
```diff
@@ -991,6 +987,7 @@ Global
 		{1A5321F3-B837-4EB6-9547-37CC70088EA9} = {C5EFA126-F329-4F8B-8DC9-E4182F4976D0}
 		{E95C2CA1-FF23-47CC-A896-CC5175B37890} = {67599AD5-51EC-44CB-85CE-B60CD8CBA270}
 		{9D23C997-1F46-444A-8C07-4A4BFF7E4E63} = {E95C2CA1-FF23-47CC-A896-CC5175B37890}
+		{80AF98CA-BC1D-4011-8460-5671799EC419} = {E95C2CA1-FF23-47CC-A896-CC5175B37890}
```
Comment (Felix-Dev) [inline MUXControlsInnerLoop.sln:990]: This is the actually relevant line for this PR here, playing the NumberBox's APITests project correctly in the project structure now.

Comment (chingucoding) [reply MUXControlsInnerLoop.sln:990]: Strangely, #3146 also introduced that line. We lost that loc with #2977 though.

**Pull request merged**